tools/clang/test/taef/lit.cfg

```diff
@@ -34,4 +34,12 @@ else:
   hlsl_data_dir = os.path.join(config.llvm_src_root, 'tools', 'clang', 'test', 'HLSL')
 
   test_dir = os.path.join(config.llvm_obj_root, config.llvm_build_mode, 'test')
-  config.test_format = lit.formats.TaefTest(config.te, test_dll, hlsl_data_dir, test_dir)
+
+  param_hlsl_data_dir = str.format('/p:HlslDataDir={}', hlsl_data_dir)
+  extra_params = [param_hlsl_data_dir]
+
+  verbose = lit_config.params.get('verbose', None)
+  if verbose == None:
+      extra_params.append('/logOutput:LowWithConsoleBuffering')
```

Comment:
Does this address the taef tests in the `unittests` directory including `CompilerTest`? I would have thought they'd need a priority parameter too as some of them are restricted based on it

Reply:
Yes. I'll add the priority parameter for taef/lit.cfg too.